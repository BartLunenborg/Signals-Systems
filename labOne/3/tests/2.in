1 42 0 42 -3.1415927
