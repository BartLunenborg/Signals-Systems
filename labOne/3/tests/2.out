x(t)=0.00
